django/db/models/expressions.py

```diff
@@ -186,6 +186,7 @@ class BaseExpression:
     set_returning = False
     # Does the expression allow composite expressions?
     allows_composite_expressions = False
+    disallowed_composite_expressions_msg = None
```

Comment:
The only reason for this attribute is to cover the specialized error message that was added for `CombinedExpression`. If we could live with `CombinedExpression does not support composite primary keys.` instead it could be removed from this patch.

Reply:
I'd be happy to unify the messages. (This was just an accident of the order in which we worked on the first batch of release blockers.)

Reply:
Sounds good, I'll rework the patch set later today.